packages/lesswrong/components/posts/PostsPage/PostsPageCrosspostWrapper.tsx

```diff
@@ -59,11 +59,11 @@ const PostsPageCrosspostWrapper = ({post, refetch, fetchProps}: {
   };
 
   if (!contextValue.hostedHere) {
-    contextValue.combinedPost = {
-      ...document,
-      ...post,
-      contents: document?.contents ?? post.contents,
-    };
+    const overrideFields = ["contents", "readTimeMinutes"];
+    contextValue.combinedPost = {...document, ...post};
+    for (const field of overrideFields) {
+      contextValue.combinedPost[field] = document?.[field] ?? post[field];
+    }
```

Comment:
I think it would be helpful to add a comment here explaining what is conceptually happening. Something like:

`For most of the posts's fields, we prefer to use our value (ex. votes, authors, comments, etc). However, for fields related to the post contents (ex. read time), we prefer the original post.`